CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Post-Processing 
```

Comment:
Hi Eric, I followed these commands when working on the remove_variants_by_annotations.py conversion to a subcommand and was able to understand the instructions clearly. Great job on explaining the steps/resources needed to set up these commands!